bedrock/sitemaps/utils.py

```diff
@@ -150,10 +157,35 @@ def get_static_urls():
     return urls
 
 
+def _get_vrc_urls():
+
+    urls = defaultdict(list)
+
+    for entry in ContentfulEntry.objects.filter(
+        content_type=CONTENT_TYPE_PAGE_RESOURCE_CENTER,
+        classification=CONTENT_CLASSIFICATION_VPN,
+    ):
+        _path = f"{VRC_ROOT_PATH}{entry.slug}"
```

Comment:
I think we should add the trailing slash here, that's the actual URL and not the redirect.

Reply:
Good point - nice spot. Done!